### PR TITLE
Implement header link style

### PR DESCRIPTION
**Pull request by gregberge**
- **Add a way to silent logs**
- **Fix active flag (to be removed)**
- **Support link header style**

**Default light**
default-light
**Default dark**
default-dark
**Bold light**
bold-light
**Bold dark**
bold-dark
**Contrast light**
contrast-light
**Contrast dark**
contrast-dark
**Custom light**
custom-light
**Custom dark**
custom-dark

### CODE REVIEW

File: packages/gitbook/src/app/(space)/layout.tsx [hunk 1]
```diff
@@ -98,6 +98,7 @@ export default async function SpaceRootLayout(props: { children: React.ReactNode
                             headerTheme.backgroundColor.light,
                         )}
                         ${generateColorVariable('header-link', headerTheme.linkColor.light)}
+                        ${generateColorVariable('header-button-text', colorContrast(headerTheme.linkColor.light as string, ['#000', '#fff']))}
```
Comment (SamyPesse) [inline packages/gitbook/src/app/(space)/layout.tsx:101]: Not for this PR: I'm just wondering if we are not starting to generate too many css variables, and how it impacts the HTML payload. Maybe if it's too big we should move this to a dynamic route to generate a `style.css`.

**Review by SamyPesse — APPROVED** [review]

**Review by SamyPesse — COMMENTED** [review]
@gregberge can you add some visual tests?

**Review by zenoachtig — APPROVED** [review]

File: packages/gitbook/src/app/(space)/layout.tsx (hunk 1 above)
Comment (gregberge) [reply packages/gitbook/src/app/(space)/layout.tsx:101]: I think it's OK right now, also Next.js becomes SPA once loaded so I guess it's fine.

**Comment by gregberge** [comment]
> @gregberge can you add some visual tests?

Yeah will do it.

**Pull request merged**